workflows/images/cellranger/README.md

```diff
@@ -0,0 +1,39 @@
+This folder contains a Dockerfile for the cellranger analysis.
+
+## Building the image
+
+In order to build this image, the cellranger archive component must be downloaded separately to comply with licensing, and should be placed in this folder.
```

Comment:
By `this folder` you mean `workflows/images/cellranger`?

Reply:
I assume this should also be my working directory for the next command.

Reply:
Yes, the folder where this file lives, and yes, this should be the working directory, will clarify.